### DISCUSSION TITLE
Full screen Window/Locked Window/Secured Window

### DISCUSSION THREAD

**Discussion by osc2nuke** [NativePHP/laravel#183] [Ideas]
Imagine running a NativePHP APP on a **touchscreen** device where visitors use your APP but you do not want them to be able to get into some administrator pages inside your App, or worse onto the operating system Desktop .... where they can gain access to (??) everything else?
I can imagine such feature can be handy in many scenarios. [ You prob all seen these ordering monitors at the entry in your local McDonalds? ]
So, for example access to a wireless connected keyboard would only open up the full APP with a specific keyboard keys combination or show a login for any authorized person what has either full control or a kind of group-access only.
Or whatever is better!

The usage reasons for this idea are basically limitless!
A reference something like "Secured Windows" here:
https://nativephp.com/docs/1/the-basics/windows
Would be awesome!!!!!!!

PS: I get eager myself to create an APP that resembles the current "popular" , 'Self Scan CheckOut' systems, but i not own any of the barcode-scanner devices to run tests and experiments.
Other Dev's who are interested in working together or can offer (loan) equipment can always contact me via ( E-Mail in profile) @osc2nuke

**Comment by mrfelipemartins** [comment]
Maybe you can achieve that with combinations like this:

```php
Window::open()
->frameless()
->resizable(false)
->closable(false)
->minimizable(false)
->fullscreen(true);
```
Since we are talking about self payment totems, user won't be able to simply touch out of the app.

**Comment by basst85** [comment]
There is also support for kiosk-mode: 
https://github.com/NativePHP/laravel/pull/68

**Comment by simonhamp** [comment]
Closing as this is implemented. Just needs to be added to the docs
